distribution/install-profiles/src/main/resources/features.xml

```diff
@@ -17,7 +17,7 @@
           xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
 
     <feature name="profile-standard" install="manual" version="${project.version}"
-    	description="This will install the standard set of DDF applications." start-level="1">
+    	description="This will install the standard set of applications." start-level="1">
```

Comment:
I think using 'DDF' in the profiles is appropriate. I would leave this file alone.